priv/repo/migrations/20170505224222_add_github_repo_and_owner_to_project.exs

```diff
@@ -0,0 +1,10 @@
+defmodule CodeCorps.Repo.Migrations.AddGithubRepoAndOwnerToProject do
+  use Ecto.Migration
+
+  def change do
+    alter table(:projects) do
+      add :github_repo, :string
+      add :github_owner, :string
```

Comment:
Yeah, reviewing the Github docs more it looks like we can use the repo name for everything we need